docs/develprocess.md

```diff
@@ -160,24 +159,47 @@ Special case: VERSION and XCPNGVERSION are always the same (example: `xcp-ng-rel
+  * next maintenance branch directly (e.g. `1.45.0-8.3`) once the upstream SRPMs have been released
 
 If for any reason we decide to release a newer version of the software as a maintenance update, then we'd create new tag and a new maintenance branch that match `UPSTREAMVERSION` (that changes) and `XCPNGVERSIONFULL` (that doesn't change)
 
-Special case: the upstream version and the XCP-ng version are always the same (example: `host-installer`). Then:
-* Tags: `vXCPNGVERSIONFULL` (`v8.2.0`)
-* Maintenance branch: `XCPNGVERSION` (`8.2`)
+Special case: if the upstream version and the XCP-ng version are always the same (example: `host-installer`), then:
+  * Tags: `vXCPNGVERSIONFULL` (`v8.2.0`)
+  * Maintenance branch: `XCPNGVERSION` (`8.2`)
 
 #### About upstream branches
 
 * If we get the sources from XS SRPMs, then we import them to a branch named `XS` and tag `XS-XSVERSIONFULL` (`XS-8.2.0`). Example: `host-installer`.
 * If we forked a git repository, we don't need to push the upstream branches or tags to our own fork. However it could be a good habit to track maintenance or hotfix branches for changes.
 
+#### Special case: `qemu-dp`
+
+`qemu-dp` both *has* an upstream git repository and at the same time it *hasn't*:
+* The SRPM's source tarball comes from upstream `qemu`.
+* Additional patches by XenServer team come from a private git repository, so all we have is patches in the SRPM.
+
+We chose to base our `qemu-dp` repository on a fork of the upstream `qemu` [repository mirror on github](https://github.com/qemu/qemu), with additional branches:
+* To track XS patches, for each XS release:
+  * Branch `UPSTREAMVERSION-XS-XSVERSION` (`2.12.0-XS-8.2`), created from the `v2.12.0` upstream tag, and patches from the SRPM applied as commits on top.
+  * Tag `vUPSTREAMVERSION-XS-XSVERSIONFULL` (`v2.12.0-XS-8.2.0`) created from the commit of the above branch that corresponds to the initial release of XSVERSIONFULL.
+* For XCP-ng maintenance, for each release:
+  * Branch `UPSTREAMVERSION-XCPNGVERSION` (`2.12.0-8.2`), branched from tag `vUPSTREAMVERSION-XS-XSVERSIONFULL` (`v2.12.0-XS-8.2.0`)
+  * We tag when we release a new version of XCP-ng: `vUPSTREAMVERSION-XCPNGVERSIONFULL` (`v2.12.0-8.2.0`). Can be identical to the XS tag.
+* For XCP-ng development (new features):
+  * **upstream first** if can be done. Not easy when there's no repo for XenServer patches.
+  * if we really need to diverge a little from upstream, on a temporary dev branch `dev-NEXTXCPNGVERSION` (e.g. `dev-8.3`), based either on the current maintenance branch, or on a branch that is likely to be the one used in the next release (we don't always know!)
+  * next maintenance branch directly (e.g. `2.12.0-8.3`) once the upstream SRPMs have been released
```

Comment:
It's duplicated but not a big section and the example for the next maintenance branch is based on qemu version numbers. I wanted to avoid having people click to refer to another section.

Reply:
fine by me